source/get-started/install-configfile.rst

```diff
@@ -3,10 +3,11 @@
 Install using clr-installer and a configuration file
 ####################################################
 
-This page explains how to install |CL-ATTR| using the clr-installer tool
-with a configuration file. The configuration file (:file:`clr-installer.yaml`)
-can be reused to duplicate the same installation configuration on additional
-machines.
+In addition to the interactive GUI and text-based modes,
+:command:`clr-installer` also supports an unattended mode where you 
+simply provide it a YAML configuration file.  
+
+This guide shows you three examples of how to use its unattended mode.  

```

Comment:
This guide shows three examples of how to use the unattended mode.

Reply:
I meant two.  fixed.